### ISSUE TITLE
Add/Review cluster system tests based on wazuh/10771

### ISSUE THREAD

**Issue by AdriiiPRodri** [wazuh/wazuh-qa#2479]
### Description

In this issue, we are going to adapt the tests related to the agent-info task and create a test for the new agent-groups task. More information: https://github.com/wazuh/wazuh/issues/10771.

Initially, the tests to be modified are the following:
- https://github.com/wazuh/wazuh-qa/tree/master/tests/performance/test_cluster/test_cluster_performance -> In this case, we have to add the new task.
- https://github.com/wazuh/wazuh-qa/tree/master/tests/system/test_cluster/test_agent_info_sync -> We need to check that this test is still working properly.
- https://github.com/wazuh/wazuh-qa/tree/master/tests/system/test_cluster/test_agent_groups_sync -> This test should be created, it should be similar to the agent-info test.

**Comment by yanazaeva** [comment]
## Issue update

We also modified the [test_integrity_sync.py](https://github.com/wazuh/wazuh-qa/tree/master/tests/system/test_cluster/test_integrity_sync) test, deactivating the test in charge of making sure the `extra-valid` files were properly forwarded to the master from the workers.

**Comment by yanazaeva** [comment]
## To Do 

List of missing points to complete once the issue https://github.com/wazuh/wazuh/issues/11800 is complete:

- [ ] Check the `agent_info_sync` test.
- [ ] Finish the `agent_groups_sync` test.

**Comment by AdriiiPRodri** [comment]
Blocked by https://github.com/wazuh/wazuh/issues/11800

**Comment by yanazaeva** [comment]
## Issue update

### Agent info sync

This test is still working properly. We can find the test output below:

<details>
<summary>Test results</summary>


```
========================================= test session starts ==========================================
platform linux -- Python 3.9.5, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /home/yanazaeva/git/wazuh-qa
plugins: testinfra-5.0.0, metadata-1.11.0, html-3.1.1
collected 2 items                                                                                      

../../tests/system/test_cluster/test_agent_info_sync/test_agent_info_sync.py ..                  [100%]

=========================================== warnings summary ===========================================
../../wazuh_qa_env/lib/python3.9/site-packages/requests/__init__.py:89
  /home/yanazaeva/git/wazuh-qa/wazuh_qa_env/lib/python3.9/site-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.26.8) or chardet (3.0.4) doesn't match a supported version!
    warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "

-- Docs: https://docs.pytest.org/en/stable/warnings.html
--------------------- generated html file: file:///docs/system_tests_issue/1.html ----------------------
=============================== 2 passed, 1 warning in 387.66s (0:06:27) ===============================
========================================= test session starts ==========================================
platform linux -- Python 3.9.5, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /home/yanazaeva/git/wazuh-qa
plugins: testinfra-5.0.0, metadata-1.11.0, html-3.1.1
collected 2 items                                                                                      

../../tests/system/test_cluster/test_agent_info_sync/test_agent_info_sync.py ..                  [100%]

=========================================== warnings summary ===========================================
../../wazuh_qa_env/lib/python3.9/site-packages/requests/__init__.py:89
  /home/yanazaeva/git/wazuh-qa/wazuh_qa_env/lib/python3.9/site-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.26.8) or chardet (3.0.4) doesn't match a supported version!
    warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "

-- Docs: https://docs.pytest.org/en/stable/warnings.html
--------------------- generated html file: file:///docs/system_tests_issue/2.html ----------------------
=============================== 2 passed, 1 warning in 382.30s (0:06:22) ===============================
========================================= test session starts ==========================================
platform linux -- Python 3.9.5, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /home/yanazaeva/git/wazuh-qa
plugins: testinfra-5.0.0, metadata-1.11.0, html-3.1.1
collected 2 items                                                                                      

../../tests/system/test_cluster/test_agent_info_sync/test_agent_info_sync.py ..                  [100%]

=========================================== warnings summary ===========================================
../../wazuh_qa_env/lib/python3.9/site-packages/requests/__init__.py:89
  /home/yanazaeva/git/wazuh-qa/wazuh_qa_env/lib/python3.9/site-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.26.8) or chardet (3.0.4) doesn't match a supported version!
    warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "

-- Docs: https://docs.pytest.org/en/stable/warnings.html
--------------------- generated html file: file:///docs/system_tests_issue/3.html ----------------------
=============================== 2 passed, 1 warning in 366.07s (0:06:06) ===============================
```

</details>

||Results|
|:--:|:--:|
|R1| [🟢 ](https://github.com/wazuh/wazuh-qa/files/8185011/1.zip) |
|R2| [🟢 ](https://github.com/wazuh/wazuh-qa/files/8185012/2.zip) |
|R3| [🟢 ](https://github.com/wazuh/wazuh-qa/files/8185013/3.zip) |
